src/components/fields/BooleanField.js

```diff
@@ -4,13 +4,17 @@ import {
   defaultFieldValue,
   getAlternativeWidget,
   optionsList,
-  getDefaultRegistry
+  getDefaultRegistry,
+  isObject,
 } from "../../utils";
 import CheckboxWidget from "./../widgets/CheckboxWidget";
 
 
-function buildOptions(schema) {
+function buildOptions(schema, uiSchema={}) {
   return {
+    inline: isObject(uiSchema) &&
+            isObject(uiSchema.options) &&
+            uiSchema.options.inline,
```

Comment:
Yeah, dunno why my sublime eslint plugin shows me this in error though :(

Anyway, the thing is the argument can be differently typed, so asuming it's always a destructurable object is highly error prone methinks (see added comments in https://github.com/mozilla-services/react-jsonschema-form/pull/348/commits/6063243fc613684a38193280080899c300f88085).